to_do.js

```diff
@@ -1,22 +1,43 @@
-const list = document.querySelector('.uList');
-debugger
-// delete item
-// list.addEventListener('click', (event) => {
-//   if(e.target.className == 'delete'){
-//     const li = e.target.parentElement;
-//     li.parentNode.removeChild(li);
-//   }
-// });
+document.addEventListener("DOMContentLoaded",()=>{
+  let list = document.querySelector(".list")
+  let form = document.querySelector(".form")
+  let enterItem = document.querySelector(".enterItem")
+  let btn = document.querySelector(".btn")
+  let error = document.querySelector(".error")
+
+  btn.addEventListener("click", () => {
+    let newItem = document.createElement('li')
```

Comment:
Good work up to this point, but you might want to check if the inputValue is valid before you append the value to  list.